x/staging/fees/keeper/keeper.go

```diff
@@ -56,6 +62,7 @@ func (k Keeper) CheckFees(ctx sdk.Context, fees sdk.Coins, msgs []sdk.Msg) error
 	}
 
 	if !requiredFees.IsZero() && (requiredFees.IsAnyGT(fees) || fees.IsZero()) {
+		k.Logger(ctx).Error("required fees not satisfied", "required", requiredFees, "given", fees)
```

Comment:
I think it's better to avoid logging these kind of errors since they will be included inside the transaction result anyway. By using the `Error` method the message will be displayed inside full nodes, which might alert node operators for nothing. We should use that only to log errors that might cause the node to crash

```suggestion
```